src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs

```diff
@@ -1055,6 +1074,20 @@ private async Task SendWithLock(ConnectionState expectedConnectionState, HubMess
                 Log.ReceivedPing(_logger);
                 // timeout is reset above, on receiving any message
                 break;
+            case AckMessage ackMessage:
+                _logger.LogInformation("Received Ack with ID {id}", ackMessage.SequenceId);
```

Comment:
Thanks for taking a look at the PR Marc 😃 

I'm updating this to Trace and using the source gen logging right now. I just put them here as Info and non-source-gen for easier debugging purposes, and since I fixed the bugs I was looking for and am doing cleanup now, this is being updated.